src/commands/createNewProject/ProjectCreateStep/JavaScriptProjectCreateStep.ts

```diff
@@ -0,0 +1,108 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ScriptProjectCreateStep } from './ScriptProjectCreateStep';
+
+export class JavaScriptProjectCreateStep extends ScriptProjectCreateStep {
+    // tslint:disable-next-line: no-use-before-declare
+    protected gitignore: string = nodeGitignore;
+
+    constructor() {
+        super();
+        this.funcignore.push('*.js.map', '*.ts', 'tsconfig.json');
+    }
+}
+
+// https://raw.githubusercontent.com/github/gitignore/master/Node.gitignore
```

Comment:
What do you think about grabbing this dynamically?  I know that there are some downsides to it, but we wouldn't have to maintain this list.

Reply:
We would have to have a backup anyways since we've seen plenty of issues with corporate proxies, etc. where pulling something dynamically from the internet doesn't always work. Doesn't seem worth it to me - but you're welcome to file an issue and we can discuss further in triage if you think it's valuable.